ads/navegg.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {loadScript} from '../3p/3p';
+import {doubleclick} from '../ads/google/doubleclick';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function navegg(global, data) {
+  const acc = data.acc;
```

Comment:
question: is `data.acc` required? what if it is undefined? If it is required, please update in the doc, also use `validateData` to valid it.